### ISSUE TITLE
Unknown layer_type ROIPooling

### ISSUE THREAD

**Issue by LiviuMarina** [ShaoqingRen/faster_rcnn#118]
Hi,

I am trying to train PASCAL_VOC model. Unfortunately,for some time, I am having this error, that the ROIPooling layer is unknown.
The roi_pooling_layer.cpp, roi_pooling_layer.cu can be found in ...src/caffe/layers.
Also roi_pooling_layer.hpp can be found in ...include/caffe/layers.

When I checked layer_factory.cpp I saw that the layer is not registered.

I have to specify that I am using caffe with windows.

Did someone encountered this issue?
Any help will be highly appreciated.

This is the error:

I0922 16:29:10.121187 10272 layer_factory.hpp:77] Creating layer roi_pool5
F0922 16:29:10.121187 10272 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: ROIPooling (known types: AbsVal, Accuray, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convoltion, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclieanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageDaa, InfogainLoss, InnerProduct, Input, LRN, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, ReLU, Reduction, Reshape, SP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, SmoothL1Loss, Softmx, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)
**\* Check failure stack trace: ***

Thank you!
